lib/vanagon/utilities/extra_files_signer.rb

```diff
@@ -19,11 +19,13 @@ def commands(project, mktemp, source_dir) # rubocop:disable Metrics/AbcSize
             remote_host = "#{project.signing_username}@#{project.signing_hostname}"
             remote_destination_path = "#{remote_host}:#{tempdir}"
             remote_file_location = "#{remote_host}:#{file_location}"
+            extra_flags = ''
```

Comment:
I don't think this line is needed

Reply:
It is, for other platforms that use the signing DSL, like windows.

Reply:
Yes, but assigning a variable like `abc = 'fgh' if something` will set the variable to nil if the condition is not satisfied, right? Tests still pass for me after removing this line.

Reply:
Yes that creates the `extra_flags` variable, I usually like to have a sane default, this keeps the code in concordance with other sides of Vanagon: https://github.com/puppetlabs/vanagon/blob/main/lib/vanagon/driver.rb#L132

Reply:
:+1: